### PR TITLE
Add register API

### PR DESCRIPTION
**Pull request by jleonoras**

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -18,6 +19,7 @@ function App() {
         <Route path="/listing" element={<Listing />} />
         <Route path="/login" element={<Login />} />
         <Route path="/register" element={<Register />} />
+        <Route path="/newregister" element={<NewRegister />} />
```
Comment (aeoxia) [inline client/src/App.js:22]: remove redundant register route

**Review by aeoxia — CHANGES_REQUESTED** [review]
Please remove the newRegister route as it is a duplicate

File: client/src/App.js (hunk 1 above)
Comment (jleonoras) [reply client/src/App.js]: Removed redundant register route.

**Review by aeoxia — APPROVED** [review]
niiice

**Pull request merged**